renovate.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ],
+  "ignorePaths": [],
+  "labels": ["Skip Changelog"],
```

Comment:
```suggestion
  "labels": ["Skip Changelog", "dependencies"],
```

Reply:
Oops, I accidentally click the apply button. @pellared I hope you are ok with this change.